### ISSUE TITLE
Ability to turn off constant snackbar notifications

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#39997]
### Description

The constant barrage of notifications and popups in Gutenerg is so annoying! Can you please add an option (saved to the database and **not** just local storage) that allows us to turn off these annoying notifications.

I don't need preview popups even time I hover over a block in the Block Inserter. I don't need to be told that I just inserted a pattern when I just clicked on a pattern to insert it. I don't need to be told that I just published a Page when I literally just clicked the Publish button.

We already get enough popups when browsing the web, we don't need them when we're simply trying to edit our own site's content. The Snackbar notifications currently show for a minimum of 20 seconds, which is waaaaay to long. Not only should this length be reduced, we should be able to turn off **ALL** of these notifications altogether.

<img width="1920" alt="screenshot_877" src="https://user-images.githubusercontent.com/2078245/161416928-8d696ecf-4b84-4e4f-b530-c84768496579.png">

<img width="1920" alt="screenshot_875" src="https://user-images.githubusercontent.com/2078245/161416556-62e22743-9655-4eaf-9472-19390c0c686a.png">

<img width="1920" alt="screenshot_876" src="https://user-images.githubusercontent.com/2078245/161416553-9c0f487b-8868-4639-8ea9-b8f11bf7f288.png">

### Step-by-step reproduction instructions

- Edit page
- Insert a new Pattern and watch the snackbar notification popup
- Publish the page and watch the snackbar notification popup
- Hover over blocks in the Block Inserter and watch the the preview popup constantly display for ever single block
- Update WP and see the Welcome message display, yet again!

### Environment info

macOS Monterey 12.3.1
Firefox 98.0.2
WP 5.9.2
Core (no plugin)

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by alexstine** [comment]
I see this being a tricky one since screen reader users may not always have the same context or UX that sighted people have. Maybe a solution is to allow the setting to hide visually without removing the screen reader alert. Just an idea to kick off the discussion. 👍

**Comment by maddisondesigns** [comment]
@alexstine I'm not sure I understand why you think this is a tricky one. I'm not suggestion to turn them off altogether. My suggestion was to provide an option to allow users to turn them off if they want to. If you're using a screen reader, have poor eyesight, or just happen to like the notifications, then you most likely wouldn't turn them off.

**Comment by alexstine** [comment]
@maddisondesigns I think the setting just needs to be clear. I might even go as far as writing something like this.

> Disable action notifications
This disables notifications for most non-important actions around the editor.
NOTE: Do not disable this if you rely on screen readers or additional accessibility tools.

Why? As I mentioned, sometimes it isn't always apparent when a certain action is completed. At the very least, I think it should be made clear to the group it would most matter to that this setting should not be used.

Thanks.

**Comment by samxmunoz** [comment]
I think you should be able to either turn off the notifications or it should be clearer that when you click them they go away.

Visual feedback like an "X" in the upper corner would be clearer that you don't need to just wait for them to disappear.

But I agree, this is a lot:

![Screen Shot 2022-07-25 at 8 25 36 AM](https://user-images.githubusercontent.com/43501707/180777262-d63e7c33-38a8-437e-949a-f3e19327f67a.png)
.
